modules/nat-gateways/output.tf

```diff
@@ -7,3 +7,8 @@ output "nat_gateway_ids" {
   description = "List of gateway IDs"
   value       = aws_nat_gateway.natgw.*.id
 }
+
+output "nat_gateway_eips" {
```

Comment:
Please name it `nat_gateway_public_ips` to better match the resource naming.

Reply:
Name updated per request.